<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>988</width>
    <height>611</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QGridLayout" name="gridLayout">
    <item row="0" column="1">
     <widget class="QGroupBox" name="groupBox">
      <property name="title">
       <string/>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout" stretch="1,1,15">
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout" stretch="1,5,2,5,2,5,3,7">
         <item>
          <widget class="QLabel" name="label">
           <property name="font">
            <font>
             <family>3ds</family>
             <pointsize>12</pointsize>
            </font>
           </property>
           <property name="text">
            <string>钢种：</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QComboBox" name="comboBox">
           <property name="font">
            <font>
             <family>3ds</family>
             <pointsize>12</pointsize>
            </font>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="label_2">
           <property name="font">
            <font>
             <family>3ds</family>
             <pointsize>12</pointsize>
            </font>
           </property>
           <property name="text">
            <string>宽度区间：</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QComboBox" name="comboBox_2">
           <property name="font">
            <font>
             <family>3ds</family>
             <pointsize>12</pointsize>
            </font>
           </property>
           <item>
            <property name="text">
             <string>700-1200</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>1200-1500</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>1500-2000</string>
            </property>
           </item>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="label_3">
           <property name="font">
            <font>
             <family>3ds</family>
             <pointsize>12</pointsize>
            </font>
           </property>
           <property name="text">
            <string>厚度区间：</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QComboBox" name="comboBox_3">
           <property name="font">
            <font>
             <family>3ds</family>
             <pointsize>12</pointsize>
            </font>
           </property>
           <item>
            <property name="text">
             <string>0.2-0.6</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>0.6-1.2</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>1.2-5.0</string>
            </property>
           </item>
          </widget>
         </item>
         <item>
          <spacer name="horizontalSpacer">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>40</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
         <item>
          <widget class="QPushButton" name="pushButton">
           <property name="font">
            <font>
             <family>3ds</family>
             <pointsize>12</pointsize>
            </font>
           </property>
           <property name="text">
            <string>加载文件</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_2" stretch="1,8,1,8,10,4,4">
         <item>
          <widget class="QLabel" name="label_4">
           <property name="font">
            <font>
             <family>3ds</family>
             <pointsize>12</pointsize>
            </font>
           </property>
           <property name="text">
            <string>宽度规格：</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLineEdit" name="lineEdit">
           <property name="font">
            <font>
             <family>3ds</family>
             <pointsize>12</pointsize>
            </font>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="label_5">
           <property name="font">
            <font>
             <family>3ds</family>
             <pointsize>12</pointsize>
            </font>
           </property>
           <property name="text">
            <string>厚度规格：</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLineEdit" name="lineEdit_2">
           <property name="font">
            <font>
             <family>3ds</family>
             <pointsize>12</pointsize>
            </font>
           </property>
          </widget>
         </item>
         <item>
          <spacer name="horizontalSpacer_2">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>40</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
         <item>
          <widget class="QPushButton" name="pushButton_2">
           <property name="font">
            <font>
             <family>3ds</family>
             <pointsize>12</pointsize>
            </font>
           </property>
           <property name="text">
            <string>显示图像</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="pushButton_3">
           <property name="font">
            <font>
             <family>3ds</family>
             <pointsize>12</pointsize>
            </font>
           </property>
           <property name="text">
            <string>一键保存图片</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <widget class="QGroupBox" name="groupBox_2">
         <property name="font">
          <font>
           <family>3ds</family>
           <pointsize>12</pointsize>
          </font>
         </property>
         <property name="title">
          <string>压下率-IU均值统计图</string>
         </property>
         <layout class="QGridLayout" name="gridLayout_3">
          <item row="0" column="0">
           <layout class="QGridLayout" name="gridLayout_2"/>
          </item>
         </layout>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
